Marlin/pins_RAMPS_FD.h

```diff
@@ -18,20 +18,20 @@
 #define X_STEP_PIN         63
 #define X_DIR_PIN          62
 #define X_ENABLE_PIN       48
-#define X_MIN_PIN          22
-#define X_MAX_PIN          2
+#define X_MIN_PIN          -1 //22
```

Comment:
Right, but the pins.h are general for all uses. You can deactivate the endstop by inverting the logic so they are for ever open. This is the easier way for the users.